src/testUtils.js

```diff
@@ -172,17 +162,27 @@ export async function destroyReport(report) {
         },
       });
 
-      await Grant.destroy({
-        where: {
-          id: grant.id,
-        },
-      });
-
-      await Recipient.destroy({
-        where: {
-          id: grant.recipientId,
-        },
-      });
+      await ActivityRecipient.findAll({ where: { grantId: grant.id } })
```

Comment:
We typically don't chain promises, can we change this to something like
```
const result = await ActivityRecipient.findAll(....)
if (results.length === 0) {
  await Grant.destroy(....)
}
```

Reply:
Sure